GOFevaluation/utils.py

```diff
@@ -577,6 +588,40 @@ def check_sample_sanity(sample):
     assert ~np.isinf(sample).any(), 'Sample contains inf values!'
 
 
+def check_for_ties(sample):
+    any_ties = len(np.unique(sample, axis=0)) != len(sample)
+    if any_ties:
+        warnings.warn("reference_sample contains ties, this might "
+                      "cause problems!", stacklevel=2)
+
+
+def check_dimensionality_for_eqpb(data_sample, reference_sample,
+                                  n_partitions, order):
+    if len(reference_sample.shape) == 1:
+        assert len(data_sample.shape) == 1, "Shape of data_sample is"\
+            " incompatible with shape of reference_sample"
+        assert isinstance(n_partitions, int), "n_partitions must be an"\
+            " integer for 1-dim. data."
+        assert order is None, "providing a not-None value for order is"\
+            " ambiguous for 1-dim. data."
+    elif len(reference_sample.shape) == 2:
+        assert len(data_sample.shape) == 2, "Shape of data_sample is"\
+            " incompatible with shape of reference_sample."
+        # Check dimensionality is two
+        assert (data_sample.shape[1]
+                == reference_sample.shape[1]
+                == len(n_partitions)), \
+            "Shape of data_sample is incompatible with shape of"\
```

Comment:
Do you think it could be useful to also check the `order` shape/dimensionality in `len(reference_sample.shape) == 2` case?